### DISCUSSION TITLE
Can you add command aliases to custom commands?

### DISCUSSION THREAD

**Discussion by TravisCarden** [ddev/ddev#6431] [Q&A] [answered]
I see that the built-in `describe` command has aliases (`describe, status, st, desc`), but I don't see a corresponding annotation for custom commands (e.g., `Aliases`). Is that functionality available?

**Comment by rfay** [comment]
I don't know of an easy way to do aliases for custom commands, but a symlink would probably work.

**Comment by stasadev** [comment, marked as answer]
See:

- https://github.com/ddev/ddev/pull/6434
